src/rprblender/export/image.py

```diff
@@ -76,15 +80,14 @@ def sync(rpr_context, image: bpy.types.Image):
         data = np.flipud(data.reshape(image.size[1], image.size[0], image.channels))
         rpr_image = rpr_context.create_image_data(image_key, np.ascontiguousarray(data))
 
-    rpr_image.set_name(image_key)
+    rpr_image.set_name(image.name + color_space)
 
     # TODO: implement more correct support of image color space types
-    if image.colorspace_settings.name in ('sRGB', 'BD16', 'Filmic Log'):
+    if color_space in ('sRGB', 'BD16', 'Filmic Log', 'SRGB'):
```

Comment:
maybe check with color_space.upper() just to not duplicate sRGB and SRGB ?

Reply:
Left the comment on why caps values are used